server/remote_cache/digest/digest.go

```diff
@@ -306,7 +307,7 @@ func HashForDigestType(digestType repb.DigestFunction_Value) (hash.Hash, error)
 	case repb.DigestFunction_BLAKE3:
 		return blake3.New(), nil
 	case repb.DigestFunction_UNKNOWN:
-		// TODO(tylerw): make this a warning when clients support this.
+		log.Warningf("Digest function was unset: defaulting to SHA256")
```

Comment:
maybe this will be too spammy still? your call.

Reply:
ReMoVeD